package.json

```diff
@@ -49,7 +49,8 @@
     "bval": "~0.1.6",
     "bweb": "~0.1.9",
     "mrmr": "~0.1.8",
-    "n64": "~0.2.10"
+    "n64": "~0.2.10",
+    "golomb": "github:tuxcanfly/golomb#bip157-updates"
```

Comment:
Based on conversation with @braydonf, we can just pull the `golomb` package into `bcoin` itself for now so that we can make progress on this branch and then in the future we can remodularize it out if `hsd` needs it.

Reply:
OK, moved to `lib` for now.